src/index.ts

```diff
@@ -181,10 +182,12 @@ export const execCompose = (
 
     childProc.stdout.on('data', (chunk): void => {
       result.out += chunk.toString()
+      if(options.callback) options.callback(chunk)
```

Comment:
[Optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) can be used instead.

```suggestion
      options.callback?.(chunk)
```